lib/pipeline.js

```diff
@@ -82,13 +82,17 @@ class PipelineModel extends BaseModel {
     formatScmUrl(scmUrl) {
         let result = scmUrl;
         const matched = (schema.config.regex.SCM_URL).exec(result);
+        let branchName = matched[4];
```

Comment:
I'm not a terribly big fan of magic numbers. It isn't your fault (since this is how it was before), but could you define this `4` as constant?